cns/cnsclient/cnsclient_test.go

```diff
@@ -227,7 +227,8 @@ func TestCNSClientRequestAndRelease(t *testing.T) {
 		t.Fatalf("Expected to not fail when releasing IP reservation found with context: %+v", err)
 	}
 
-	ipaddresses, err := cnsClient.GetIPAddressesMatchingStates(cns.Available)
+	ipaddresses, err := cnsClient.GetIPAddressesMatchingStates("Available")
+	fmt.Println(ipaddresses)
```

Comment:
this is just a test, but typically we'd want to handle the error right after it's given and then work on the response

also could use cns.Available for the free const in `cnsClient.GetIPAddressesMatchingStates` :)

Reply:
ack, changed this.